doc/nrf/releases_and_maturity/migration/migration_guide_2.8.rst

```diff
@@ -33,6 +33,14 @@ Samples and applications
 
 This section describes the changes related to samples and applications.
 
+nRF54L Series
+-------------
+
+.. toggle::
+
+   * Use the :ref:`ZMS (Zephyr Memory Storage) <zephyr:zms_api>` file subsystem for all devices that contain RRAM, such as the nRF54L Series devices.
```

Comment:
file subsystem => either use file system or storage system. I prefere the word storage as we don't handle any files in ZMS.
I also don't like the expression : "that contain RRAM".
Maybe use someting like " devices with RRAM memory technology"